os-fpga/FOEDAG
# [RG-270] Add log file for Bitstream task

> ### Motivate of the pull request
> - [x] To address an existing issue. If so, please provide a link to the issue: RG-270
> - [ ] Breaking new feature. If so, please describe details in the description part.

> ### Describe the technical details
> #### What is currently done? (Provide issue link if applicable)
> - The bitstream task was already logging to vpr_stdout.log, but the file wasn't renamed/copied after the task finished
>
> #### What does this pull request change?
> - Scope guard w/ copyLog added to Bitstream task

> ### Which part of the code base require a change
> 
> - [x] Library: CompilerOpenFPGA
> - [ ] Plug-in: <Specify the plugin name>
> - [ ] Engine
> - [ ] Documentation
> - [ ] Regression tests
> - [ ] Continous Integration (CI) scripts

> ### Testing
> - Manually tested in Raptor with the following steps:
>   - build/bin/raptor --script tests/Testcases/oneff/raptor.tcl
>   - In the task window
>     - R-Click Bitstream Generation
>     - Ensure "View Bitstream Generation Logs" is enabled and opens the log when clicked
